app/components/edit/BuyRouteEdit.tsx

```diff
@@ -49,9 +51,12 @@ const BuyRouteEdit = ({
   const [countries, setCountries] = useState<Country[]>([]);
 
   useEffect(() => {
+    const buyTypePreselection = getBuyTypePreselection();
+    if (buyTypePreselection) setValue("type", buyTypePreselection);
+
```

Comment:
Can we use form defaultValues instead?

Reply:
tried it yesterday and today, but for some reason it doesn't work. Maybe because of its conditional nature.

Reply:
Ok 🤔